### ISSUE TITLE
Add Windows 11 queues for AppCompat

### ISSUE THREAD

**Issue by ilyas1974** [dotnet/dnceng#4714]
Create Windows 11 queues and images for the AppCompat team. These queues should have the same software are the currently AppCompat queues - Windows.10.Amd64.CoreAppCompat and Windows.10.Amd64.CoreAppCompat.Pr - just a new base OS (Windows 11)

### Release Note Category
- [ ] Feature changes/additions 
- [ ] Bug fixes
- [ ] Internal Infrastructure Improvements

### Release Note Description

Add Windows 11 queues for AppCompat

**Comment by chcosta** [comment]
Working on this https://dev.azure.com/dnceng/internal/_git/dotnet-helix-machines/pullrequest/46723

Currently, even on windows 11, artifact validation hangs on the windows-new-edge validation step

> 2025-01-25 03:20:01Z -> DEBUG:   14+  >>>> & "$edgePath" --headless --enable-logging --disable-gpu --disable-gpu-compositing | Out-String

This is tracked in issue #3980 .  Now, when trying to use `--headless=old`, i get an "interesting" error message...

> 2025-01-27 20:16:45Z windows.2022.amd64.coreappcompat, windows.2022.amd64.coreappcompat.pr ==> Artifact Failure: any-os-validate: Old Headless mode has been removed from the Chrome binary. Please use the new Headless mode (https://permanently-removed.invalid/docs/chromium/new-headless) or the chrome-headless-shell which is a standalone implementation of the old Headless mode (https://permanently-removed.invalid/blog/chrome-headless-shell).

i'm now going to try `--headless=new`.  If that still fails, we should consider implementing the chrome-headless-shell artifact.

**Comment by v-zheyan** [comment]
The latest windows-new-edge-driver version is 133.0.3065.92, I revert it back to 131.0.2903.70 as the queue is not be monitored.

**Comment by Junjun-zhao** [comment]
@chcosta  Is there any update or schedule when the new Windows 11 queues will be rollout？

**Comment by ilyas1974** [comment]
Taking over this issue. Will update with the latest status later today

**Comment by Junjun-zhao** [comment]
Hi @garath , could you please let us know what's the latest plan for this work?  Since Windows 10 will reach end of support on October 14, 2025, could you please work this on priority? Because it will also require additional time for us to migrate the test from Windows 10 to Windows 11 once the queue is available.  Thank you very much!

**Comment by Junjun-zhao** [comment]
@garath Thank you for your update and continued support. I'm following up again to check if there are any updates.

We have ~5,300 test cases that need to be migrated to Windows 11 from Windows 10 21H2, which is a significant workload for our team.
Just to confirm—if we’re unable to complete the transition before Windows 10 reaches end of support, is there any workaround that would allow us to continue testing on Windows 10, such as enable the Extended Security Update (ESU) program to extend the migration time?

CC: @TaraOverfield @HeathAr

**Comment by garath** [comment]
> is there any workaround that would allow us to continue testing on Windows 10, such as enable the Extended Security Update (ESU) program to extend the migration time?

Yes, we can work with you on deadlines like that. I do not believe there are plans yet to remove this version of windows.

**Comment by davfost** [comment]
I have created a AzDo work item and assigned this to copilot to see if it can handle this type of update:

https://dev.azure.com/dnceng/internal/_workitems/edit/8277
